style/index.less

```diff
@@ -0,0 +1,18 @@
+.content {
+  margin: 20px 10%;
+
+  [data-wikipedia-preview] {
```

Comment:
According to the documentation, the [selector is configurable](https://github.com/wikimedia/wikipedia-previews#options-of-the-init-function). 

We could attach this style to another class, then apply this class in the init function when we discover all the element that should have a popup attached to them.

Reply:
Good call, I'm applying it now with [pure javascript](https://stackoverflow.com/questions/6787383/how-to-add-remove-a-class-in-javascript) since `classList` is [not supported in IE](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Browser_compatibility)